test/functional/wallet_migration.py

```diff
@@ -618,6 +618,26 @@ def test_wallet_with_relative_path(self):
         assert_equal(info["descriptors"], False)
         assert_equal(info["format"], "bdb")
 
+    def test_wallet_with_path_ending_in_slash(self):
+        self.log.info("Test migrating a wallet with a name/path ending in '/'")
+
+        # The last directory in the wallet's path
+        final_dir = "mywallet"
+        wallet_name = f"path/to/{final_dir}/"
+        wallet = self.create_legacy_wallet(wallet_name)
+        default = self.master_node.get_wallet_rpc(self.default_wallet_name)
+
+        addr = wallet.getnewaddress()
+        txid = default.sendtoaddress(addr, 1)
+        self.generate(self.master_node, 1)
+        bals = wallet.getbalances()
+
+        _, wallet = self.migrate_and_get_rpc(wallet_name)
```

Comment:
In 146b133697f07c8dda004d60087505b2e1ec14b9 "test: Migration of a wallet ending in `/`"

Could add a check for the expected backup path.

Reply:
I add this check to `migrate_and_get_rpc()` in 55e60bd9c76:

https://github.com/bitcoin/bitcoin/blob/9fa5480fc4c46fa11345c45fbe4dd21c127e3e05/test/functional/wallet_migration.py#L138-L146